src/main/java/com/farao_community/farao/gridcapa/data_bridge/configuration/DataBridgeConfiguration.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa.data_bridge.configuration;
```

Comment:
Many classes (if not all) in you `configuration` package look like they could be defined as Records. Did you try it? Does it work?

Reply:
done for files meta data